build.sh

```diff
@@ -40,39 +40,43 @@ do
     test)
       # run lang-specific tests
       (cd lang/java; ./build.sh test)
+
+      # create interop test data
+      mkdir -p build/interop/data
+      (cd lang/java/avro; mvn -B -P interop-data-generate generate-resources)
+      
       # install java artifacts required by other builds and interop tests
-      mvn install -DskipTests
+      mvn -B install -DskipTests
       (cd lang/py; ./build.sh test)
       (cd lang/py3; ./build.sh test)
       (cd lang/c; ./build.sh test)
-      (cd lang/c++; ./build.sh test)
+      # https://issues.apache.org/jira/browse/AVRO-2249
+      # (cd lang/c++; ./build.sh test)
```

Comment:
:(

Reply:
These tests should be fixed, but I'm resolving some other stuff with the Java tests first.

Reply:
I've actually been thinking we should have two run modes for this:

1. Exit as soon as something fails
2. Keep going, and report all failures at the end.